.github/workflows/python.yml

```diff
@@ -57,16 +57,22 @@ jobs:
             fail-fast: false
             matrix:
                 engine: ${{ fromJson(needs.load-engine-matrix.outputs.matrix) }}
+                python:
+                  - "3.8"
+                  - "3.9"
```

Comment:
So, do we support 3.8 and 3.9 or not?
I still hear differnt gossips even now, after this PR merged.